src/tests/functional/plugin/gpu/shared_tests_instances/multi/gpu_remote_blob_tests.cpp

```diff
@@ -108,7 +108,10 @@ auto device_names_and_support_for_remote_blobs2 = []() {
     return std::vector<DevicesNamseAndProperties>{
 #ifdef ENABLE_INTEL_CPU
         //{{CPU}, {}},  // stand-alone CPU via MULTI (no GPU), no OCL context
-        {{CPU}, {ov::intel_auto::device_bind_buffer(true)}},  // stand-alone CPU via MULTI (no GPU), no OCL context
+        {{CPU, "GPU.1"},
```

Comment:
What if we don't have `GPU.1`?? 
GPU.0 is default

Reply:
Add the comments to the code to explain the logic here.